Tasks/JenkinsDownloadArtifacts/jenkinsdownloadartifacts.ts

```diff
@@ -80,25 +25,17 @@ async function doWork() {
         const serverEndpointAuth: tl.EndpointAuthorization = tl.getEndpointAuthorization(serverEndpoint, false);
         const username: string = serverEndpointAuth['parameters']['username'];
         const password: string = serverEndpointAuth['parameters']['password'];
-        const cred: Credential = (username && password) ? new Credential(username, password) : new Credential('', '');
 
         const jobName: string = tl.getInput('jobName', true);
         const localPathRoot: string = tl.getPathInput('saveTo', true);
 
         const strictSSL: boolean = ('true' !== tl.getEndpointDataParameter(serverEndpoint, 'acceptUntrustedCerts', true));
 
-        const jenkinsBuild: string = tl.getInput('jenkinsBuild', true);
-        let buildId: number;
-        if (jenkinsBuild === 'LastSuccessfulBuild') {
-            tl.debug(tl.loc('GetArtifactsFromLastSuccessfulBuild', jobName));
-            buildId = await getLastSuccessful(serverEndpointUrl, jobName, cred, strictSSL);
-        } else {
-            const buildIdStr: string = tl.getInput('jenkinsBuildNumber');
-            buildId = parseInt(buildIdStr);
-        }
-        tl.debug(tl.loc('GetArtifactsFromBuildNumber', buildId, jobName));
+        let jenkinsClient: JenkinsRestClient = new JenkinsRestClient();
+        let jenkinsJobDetails: JenkinsJobDetails = await jenkinsClient.GetJobDetails();
 
-        const artifactQueryUrl: string = `${serverEndpointUrl}/job/${jobName}/${buildId}/api/json?tree=artifacts[*]`;
+        const artifactQueryUrl: string = `${serverEndpointUrl}/job/${jobName}${jenkinsJobDetails.multibranchPipelineUrlInfix}/${jenkinsJobDetails.buildId}/api/json?tree=artifacts[*]`;
+        tl.debug(tl.loc('GetArtifactsFromBuildNumber', jenkinsJobDetails.buildId, jobName));
```

Comment:
debug doesn't require localized string. make it console.log?

Reply:
changed.